mono/sgen/sgen-cardtable.c

```diff
@@ -441,18 +441,26 @@ sgen_card_table_clear_cards (void)
 }
 
 static void
-sgen_card_table_start_scan_remsets (void)
+sgen_card_table_start_scan_remsets (gboolean is_parallel, SgenObjectOperations *object_ops_nopar, SgenObjectOperations *object_ops_par)
```

Comment:
Why do we pass object operations here ? Confused me for a second. If we want to be strict about it ParallelIterateBlockRangesJob shouldn't inherit from ParallelScanJob since it doesn't scan any objects. We could make it inherit from the basic SgenThreadPoolJob and remove all references of object_ops from this PR

Reply:
Yes, I pretty much just followed pattern for similar logic when doing it, but agree that we should change it and drop object_ops since this is only used when parallelizing anyways.